here-naksha-lib-handlers/src/main/java/com/here/naksha/lib/handlers/IntHandlerForEventHandlers.java

```diff
@@ -57,4 +66,22 @@ public IntHandlerForEventHandlers(final @NotNull INaksha hub) {
       return notImplemented(event);
     }
   }
+
+  private @NotNull Result validateWriteRequest(final @NotNull WriteRequest<?, ?> wr) {
+    Result result = null;
+    for (final WriteOp<?> wOp : wr.queries) {
+      final EventHandler eventHandler = (EventHandler) wOp.feature;
+      // Common Plugin specific validations
+      result = validateWritePluginRequest(eventHandler);
+      if (result instanceof ErrorResult) return result;
+      // TODO : handler specific validations in future, as needed
+      if (eventHandler.getClassName().equals("com.here.naksha.lib.handlers.DefaultStorageHandler")) {
+        final Object storageId = eventHandler.getProperties().get(EventHandlerProperties.STORAGE_ID);
+        if (storageId == null || storageId.toString().isEmpty())
+          return new ErrorResult(XyzError.ILLEGAL_ARGUMENT, "Mandatory parameter properties.storageId missing!");
+        // TODO check if storageId is valid
```

Comment:
nth / future idea: we might consider adding ticket numbers along committed TODOs to make them more addressable ;)

Reply:
agree ... for this TODO, we would likely address it sooner and not very far in future ... so better add ticket to the same Epic (MCPODS-6178) and put it as the last number in the list. Right now I see 52 tasks already in the list, so this will go to #53

Reply:
great, I added the ticket number to the TODO